libraries/AP_HAL_ChibiOS/hwdef/scripts/defaults_periph.h

```diff
@@ -304,7 +304,7 @@
 #define AP_PERIPH_IMU_ENABLED 0
 #endif
 #ifndef AP_PERIPH_RC_OUT_ENABLED
-#define AP_PERIPH_RC_OUT_ENABLED 0
+#define AP_PERIPH_RC_OUT_ENABLED AP_PERIPH_NOTIFY_ENABLED
```

Comment:
This doesn't look right?!

Reply:
`AP_PERIPH_NOTIFY_ENABLED` depends on `AP_PERIPH_RC_OUT_ENABLED`. I was just trying to get the failing builds passing. Can we not magically turn the `AP_PERIPH_RC_OUT_ENABLED ` unless explicitly disabled?

Reply:
> `AP_PERIPH_NOTIFY_ENABLED` needs `AP_PERIPH_RC_OUT_ENABLED`. I was just trying to get the failing builds passing. Can we not magically turn the `AP_PERIPH_RC_OUT_ENABLED ` ON unless explicitly disabled?

Not on periph :-)

*Why* does Notify require RC_OUT?  Notify significantly predates RC out, I think

Reply:
https://github.com/ardupilot/ardupilot/blob/master/Tools/AP_Periph/AP_Periph.h#L82 

This is what confused me. Either we have to enable RC_OUT or explicitly tell it to work without it.

Reply:
Alright! I'll find something else. Thanks!

Reply:
We've now removed the sanity check as bogus, so might be able to proceeed with this one!

Reply:
Yep, updated and pushed. Gave it a try on CBS, works fine as expected.